src/pystatis/config.py

```diff
@@ -32,6 +32,34 @@
     "zensus": re.compile(r"^\d{4}[A-Z]-\d{4}$"),
     "regio": re.compile(r"^((\d{5}-.{1,2}($|-.*$))|(A.*$)|([0-9A-Z]{10}$))"),
 }
+COLUMN_NAME_DICT = {
+    "genesis": {
+        "de": {
+            "time_label": "Zeit_Label",
+            "time": "Zeit",
+            "variable_code": "Auspraegung_Code",
```

Comment:
we should stick with the official vocabulary: "Merkmal" is a variable and "Ausprägung" is the value for a variable, so I would expect "Auspraegung_Code" to be "value_code", not "variable_code" etc.

Reply:
As far as I can tell the term "value" is used for the actual numeric value columns and the value_label is the name of that numeric value. "Merkmal" is the variable but "Ausprägung" does not seem to have a consistent English name (or at least I couldn't identify it). In Genesis/Regionalstatistik the respective columns are referred to as variable_label.2 and in Zensus, they seem to be called attributes (variable_attribute_label). I called them "levels" to reflect the (typically nominal) scaling levels of the corresponding variable. So e.g. (variable_label = Bundesland, variable_level = Niedersachsen, value_label = Personen, value = 123). I could change "level" to "attribute" if you think that's more in keeping with the genesis/zensus naming conventions (although I don't think attribute is the right term for what the respective columns actually contain..)?

Reply:
Switched the keys to "value" instead of "variable_level" to be more generic (the variables reflect Months etc. as well).